client/state/data-layer/wpcom/privacy-policy/index.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * External dependencies
+ *
+ * @format
+ */
+
+/**
+ * Internal dependencies
+ */
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
+import { PRIVACY_POLICY_REQUEST } from 'state/action-types';
+import { privacyPolicyReceive } from 'state/privacy-policy/actions';
+
+/*
+ * Start a request to WordPress.com server to get the privacy policy data
+ */
+export const fetchPrivacyPolicy = ( { dispatch }, action ) =>
+	dispatch(
+		http(
+			{
+				method: 'GET',
+				path: '/privacy-policy',
+				apiNamespace: 'wpcom/v2',
+			},
+			action
+		)
+	);
+
+export const addPrivacyPolicy = ( { dispatch }, action, data ) =>
+	dispatch( privacyPolicyReceive( data ) );
```

Comment:
This would be a good spot to manipulate any client data. For example passing through `data.entities.automattic` instead